lib/util/logger.js

```diff
@@ -2,11 +2,49 @@
 
 const chalk = require('chalk');
 const figlet = require('figlet');
+const winston = require('winston');
+const DailyRotateFile = require('winston-daily-rotate-file');
+
+winston.configure({
+  exitOnError: false,
+  transports: [
+    new DailyRotateFile({
+      level: 'debug',
+      showLevel: true,
+      handleExceptions: true,
+      humanReadableUnhandledException: true,
+      filename: 'markbind-%DATE%.log',
+      datePattern: 'YYYY-MM-DD',
+      dirname: '_markbind/logs',
+      maxFiles: '2d',
+    }),
+  ],
+});
 
 module.exports = {
-  info: text => console.log(chalk.cyan('info: ') + text),
-  warn: text => console.log(chalk.yellow(`warning: ${text}`)),
-  error: text => console.log(chalk.red(`error: ${text}`)),
-  log: text => console.log(text),
+  info: (text) => {
+    console.log(chalk.cyan('info: ') + text);
+    winston.info(text);
+  },
+  warn: (text) => {
+    console.log(chalk.yellow(`warning: ${text}`));
+    winston.warn(text);
+  },
+  error: (text) => {
+    console.log(chalk.red(`error: ${text}`));
+    winston.error(text);
+  },
```

Comment:
- Let's reorder these in descending severity (according to `winston`):
  - error
  - warn
  - info